docs/getting-started.md

```diff
@@ -552,6 +546,19 @@ histogram of ages, using either four lines of Python or just a few more lines of
     )
     ```
 
+=== "Stata"
+
+    1. Right-click on the `analysis` folder in the editor's Explorer and select
+       "New file". Type "report.do" as the filename and press ++enter++.
+    2. Add the following to `report.do`:.
+    ```stata
+    !gunzip output/input.csv.gz
+    import delimited using output/input.csv
+
+    // TODO plot equivalent
+    // TODO save plot
```

Comment:
question: Do these TODOs need addressing?

Reply:
Yeah, but am going to do a follow on to address